services/ui_backend_service/.gitignore

```diff
@@ -0,0 +1,2 @@
+# user-defined configurations
+links.json
```

Comment:
these can be cleaned up, as custom links config was moved to be done with an ENV variable

Reply:
cleaned up in #165